.rubocop.yml

```diff
@@ -62,9 +65,15 @@ Style/EachWithObject:
 Style/Encoding:
   Enabled: false
 
+Style/EmptyLinesAroundClassBody:
```

Comment:
Yeah, I know. But the case it complained about was wrong IMO, which was why I disabled it.